src/main/java/com/solr/clientwrapper/infrastructure/repository/EmployeeRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.solr.clientwrapper.infrastructure.repository;
+
+
+import com.solr.clientwrapper.infrastructure.entity.Employee;
+import org.springframework.data.solr.repository.Query;
+import org.springframework.data.solr.repository.SolrCrudRepository;
+
+import java.util.List;
+
+
+public interface EmployeeRepository extends SolrCrudRepository<Employee, Integer> {
```

Comment:
Why is this "employee" collection hardcoded...

Reply:
employee related class created you only

Reply:
I think you have created your branch from the older main branch. Please pull the new main branch and merge the main to your branch. (Do not merge your branch to main branch)